integration/mediation-tests/tests-mediator-1/src/test/resources/artifacts/ESB/server/repository/deployment/server/synapse-configs/default/proxy-services/CallMediatorReplyNoDefaultContentTypeTestProxy.xml

```diff
@@ -0,0 +1,15 @@
+<proxy name="CallMediatorReplyNoDefaultContentTypeTestProxy"
+       startOnLoad="true" trace="disable" transports="http https">
+<description/>
+<target>
+    <inSequence>
+        <log level="custom">
+            <property expression="get-property('transport','Content-Type')" name="Default_ContentType_Test_ContentType"/>
+        </log>
+        <drop/>
+    </inSequence>
+    <outSequence>
```

Comment:
remove the outSequence. cause after drop it not get called

Reply:
Sure, Will do